one/src/daemon.rs

```diff
@@ -274,6 +274,16 @@ pub struct DaemonOpts {
         default_value = "file://./pipeline"
     )]
     object_store_url: url::Url,
+
+    /// Comma-separated list of stream IDs or model IDs to register as initial interests on startup.
+    /// The node will track these streams and sync their data.
+    #[arg(
+        long,
+        use_value_delimiter = true,
+        value_delimiter = ',',
+        env = "CERAMIC_ONE_INITIAL_INTERESTS"
+    )]
+    initial_interests: Vec<String>,
```

Comment:
nit: trying to decide what this name should be.. it's not exactly "initial" because they don't ever go away, but _NEW_ or _MODEL_ steams a bit silly

Reply:
also, I wonder if we want to subscribe to the `METAMODEL_STREAM_ID` automatically/easily? unless we do and I missed it, but that's the "I'd like to see all models on the network".

Reply:
`EXTRA_INTERESTS` kinda matches how the daemon treats peers, so I'll go with that :ok_hand: